consensus/src/liveness/round_state.rs

```diff
@@ -278,10 +278,15 @@ impl RoundState {
         self.vote_sent.clone()
     }
 
+    /// Setup a longer timeout task for leader because it enters the round earlier.
+    pub fn setup_leader_timeout(&mut self) {
```

Comment:
Do you think having this configurable might be handy?

Reply:
I can add it, but doesn't feel we'll need to change this often though